src/index.ts

```diff
@@ -77,7 +77,7 @@ namespace outputWrapper {
     statSync: typeof statSync,
     writeFileSync: typeof writeFileSync,
     appendFileSync: typeof appendFileSync,
-    rmdirSync: typeof rmdirSync,
+    rmdirSync: (path: string, options?: { recursive?: boolean; }) => void,
```

Comment:
This LGTM.

If this is an interim solution, we may want to leave a comment saying when it will be restored to` typeof rmdirSync`, so we don't forget.

Reply:
That's a good point. I have added a `TODO` comment